addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/chime.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="rfxcom" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="chime">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge" />
+			<bridge-type-ref id="tcpbridge" />
+			<bridge-type-ref id="RFXtrx433" />
+			<bridge-type-ref id="RFXrec433" />
+		</supported-bridge-type-refs>
+
+		<label>RFXCOM Chime</label>
+		<description>A Chime device.</description>
+
+		<channels>
+			<channel id="chimeSound" typeId="chimesound" />
+			<channel id="signalLevel" typeId="system.signal-strength" />
```

Comment:
Should we also included a contact because not all chimes support a sound but I would assume we want to receive some trigger of them anyway.